### PR TITLE
Rename repo/package `mom6_regional` -> `regional_mom6`

### PR DESCRIPTION
**Pull request by navidcy**
Last time. :)

### CODE REVIEW

File: demos/model-forced.ipynb [hunk 1]
```diff
@@ -453,10 +453,10 @@
     "import subprocess\n",
     "from scipy.ndimage import binary_fill_holes\n",
     "from importlib import reload\n",
-    "# os.chdir(\"/path/to/where/you/cloned/the/mom6-regional/repository\")\n",
-    "os.chdir(\"/home/149/ab8992/cosima_regional/mom6-regional-scripts/mom6_regional/\")\n",
+    "# os.chdir(\"/path/to/where/you/cloned/the/regional-mom6/repository\")\n",
+    "os.chdir(\"/home/149/ab8992/cosima_regional/regional-mom6/regional_mom6/\")\n",
     "\n",
-    "import mom6_regional as ml\n",
+    "import regional_mom6 as ml\n",
```
Comment (angus-g) [inline demos/model-forced.ipynb:459]: I guess per the discussion this should be:
```suggestion
    "import regional_mom6 as rm\n",
```

File: demos/reanalysis-forced.ipynb [hunk 2]
```diff
@@ -331,10 +331,10 @@
     "import subprocess\n",
     "from scipy.ndimage import binary_fill_holes\n",
     "from importlib import reload\n",
-    "# os.chdir(\"/path/to/where/you/cloned/the/mom6-regional/repository\")\n",
-    "os.chdir(\"/home/149/ab8992/cosima_regional/mom6-regional-scripts/mom6_regional/\")\n",
+    "# os.chdir(\"/path/to/where/you/cloned/the/regional-mom6/repository\")\n",
+    "os.chdir(\"/home/149/ab8992/cosima_regional/regional-mom6/regional_mom6/\")\n",
     "\n",
-    "import mom6_regional as ml\n",
+    "import regional_mom6 as ml\n",
```
Comment (angus-g) [inline demos/reanalysis-forced.ipynb:337]: And here

```suggestion
    "import regional_mom6 as rm\n",
```

**Review by angus-g — CHANGES_REQUESTED** [review]
I think this should be fine assuming all the automation still works!

**Comment by navidcy** [comment]
the notebook will need an update to replace all `ml.` to `rm.`... I'll go through them now :)

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 
-`regional_mom6` contains functions and classes that do most of the legwork in setting up a regional domain in MOM6.
-Just provide it with some information about where, when and how big, and point it in the direction of your raw input files, and it'll sort out the boring details to create MOM6-friendly input files. Check out the [demo notebook](https://nbviewer.org/github/COSIMA/regional-mom6/blob/main/demo.ipynb)!
+Just provide some information about where, when and how big, and also where raw input forcing files live. The package will sort out all the boring details to create MOM6-friendly input files.
+
+Check out the [demo notebook](https://nbviewer.org/github/COSIMA/regional-mom6/blob/m/demo.ipynb) and also the [documentation](https://readthedocs.org/projects/regional-mom6/badge/?version=latest)!
```
Comment (angus-g) [inline README.md:7]: Oops, link broke:

```suggestion
Check out the [demo notebook](https://nbviewer.org/github/COSIMA/regional-mom6/blob/main/demo.ipynb) and also the [documentation](https://readthedocs.org/projects/regional-mom6/badge/?version=latest)!
```

**Comment by navidcy** [comment]
there is a chance I broke the docs?

**Comment by angus-g** [comment]
It should be coming back online, I had to fiddle some stuff because of the name change.

**Comment by navidcy** [comment]
Where do you fiddle?

**Comment by angus-g** [comment]
It's set up on readthedocs.org, I think it should all be working now though.

**Comment by navidcy** [comment]
The codecov is OK with the name change ;)

**Comment by navidcy** [comment]
I think this is good to go.

**Comment by angus-g** [comment]
Nearly! Formatting failure, then hopefully tests pass.

**Review by angus-g — APPROVED** [review]
Seems all good now!

**Comment by navidcy** [comment]
All goof :)

**Pull request merged**